wpilibj/src/athena/java/edu/wpi/first/wpilibj/PWM.java

```diff
@@ -325,7 +235,15 @@ public void setRaw(int value) {
    * @return Raw PWM control value. Range: 0 - 255.
```

Comment:
Can wait to a later cleanup, but clearly doesn't return 0-255 anytime recently..